scrapers/cscd.py

```
@@ -0,0 +1,134 @@
+import scraper as ts
+blacklisted_sections = ["Deconfirmed/Removed/Moved Towers", "Miscellaneous Towers"]
+ts.init("cscd", "cscdroblox", ["Scope_1", "Scope_2", "Scope_3", "The_Nil_Zone"], [], blacklisted_sections, [])
+
+file = open("public/static/data/games/cscd.txt", "w")
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)